components/autofill/src/db/credit_cards.rs

```diff
@@ -95,7 +95,7 @@ pub fn get_all_credit_cards(conn: &Connection) -> Result<Vec<InternalCreditCard>
         let mut stmt = tx.prepare(&sql)?;
         credit_cards = stmt
             .query_map(NO_PARAMS, InternalCreditCard::from_row)?
-            .collect::<Result<Vec<InternalCreditCard>, _>>()?;
+            .collect::<std::result::Result<Vec<InternalCreditCard>, _>>()?;
```

Comment:
You might be able to make this work using `.map(Into::into).collect` rather than using `std::result::Result`, but I think the `std::result::Result` is also fine. Thanks for the explanation over in the other PR.

Reply:
I can't make that work for reasons I kindof expected given my limited understanding of some of this.

Note that it's a vec of results - so we have the same problem - we need an `Into()` for `Result<InternalCreditCard, rusqlite::Error>` whereas IIUC, we only have it for the error.

```
98 |             .map(Into::into).collect::<Result<Vec<InternalCreditCard>>>()?;
   |                  ^^^^^^^^^^ the trait `std::convert::From<std::result::Result<db::models::credit_card::InternalCreditCard, rusqlite::Error>>` is not implemented for `std::result::Result<db::models::credit_card::InternalCreditCard, error::Error>`
```

(TBH though, I don't really understand *why* it can't do that - but IIUC, that's the same reason I mentioned in the other issue about why you need to do `fn()?; Ok(())` rather than just `fn()`